eng/native/init-distro-rid.sh

```diff
@@ -138,7 +138,8 @@ initDistroRidGlobal()
         local distroRid=""
 
         # Check for musl-based distros (e.g Alpine Linux, Void Linux).
-        if "${rootfsDir}/usr/bin/ldd" --version 2>&1 | grep -q musl; then
+        if "${rootfsDir}/usr/bin/ldd" --version 2>&1 | grep -q musl ||
+           strings "${rootfsDir}/usr/bin/ldd" 2>&1 | grep -q musl; then
```

Comment:
(Nit) This looks like a missing space mistake on first glance... but I suppose it's column alignment?

I don't really have any specific suggestion--I don't know if this is considered readable in same-line-brace styles like we're using here. 😕 I have seen double-indent used on the continuation line to clarify this situation.

Reply:
Yup, it was to align it with start of condition; changed to double aligned (4+4 spaces) with respect to the `if` token.